enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -915,7 +921,10 @@
 
 (deftest pivot-query-test
   ;; sample-dataset doesn't work on Redshift yet -- see #14784
-  (mt/test-drivers (disj (mt/normal-drivers-with-feature :foreign-keys :nested-queries :left-join) :redshift)
+  (mt/test-drivers (disj
+                     (mt/normal-drivers-with-feature :foreign-keys :nested-queries :left-join)
+                     :redshift
+                     :presto-jdbc)
```

Comment:
You should make a note of why we're not running this test for `:presto-jdbc`, similar to the note about why we're not running it for Redshift above